adminpages/reports/sales.php

```diff
@@ -403,18 +413,38 @@ function pmpro_ShowMonthOrYear()
 		function drawVisualization() {
 
 			var data = google.visualization.arrayToDataTable([
-				[
-					'<?php echo esc_html( $date_function );?>', '<?php _e( 'Renewals', 'paid-memberships-pro' );?>', '<?php echo esc_html( sprintf( __( 'New %s', 'paid-memberships-pro' ), ucwords( $type ) ) );?>', '<?php _e( 'Average*', 'paid-memberships-pro' );?>', 
-				],
+				<?php echo wp_json_encode( $chart_headers ); ?>,
```

Comment:
I love this, it would be fantastic to do the same thing for the data output from $cols foreach below this. Essentially building the array of all data to pass into `arrayToDataTable( ..json encoded data.. )`.

Reply:
I've pushed some changes to move the PHP out of the JS